src/Serilog.Settings.Configuration/ConfigurationLoggerConfigurationExtensions.cs

```diff
@@ -21,6 +21,22 @@
 
 namespace Serilog
 {
+    /// <summary>
+    /// Defines the interpretation of a null DependencyContext configuration parameter.
+    /// </summary>
+    public enum ConfigurationAssemblySource
```

Comment:
This could go under the `Serilog.Configuration` namespace without too much inconvenience.